docs/api/modules/Bundling-Modules.md

```diff
@@ -0,0 +1,186 @@
+
+ ``` sh
+
+ export NODE_ENV=development
+
+ ```
+
+When the [build commands `npm run build`](#build-commands) is executed on the above root folder the following files are generated within the module. The root folder of the build would be the version number provided within the package.json.
+
+Module:
+```
+
+module
+├── README.md
+├── package.json
+├── src
+|   ├── components
+|   |    └── <NameOfComponent.jsx>
+|   |
+|   └── csp.js
+|   └── index.js
+├── build
+|   ├── <semantic-version-number>
+|        └── <module-name>.browser.js
+|        └── <module-name>.browser.js.map
+|        └── <module-name>.legacy.browser.js
+|        └── <module-name>.legacy.browser.js.map
+|        └── <module-name>.node.js
+|        └── <module-name>.node.js.map
+```
+
+### Generated files
+
+| File Name       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `<module-name>.browser.js `| This build is loaded on modern browsers. We use [babel-preset-amex](https://github.com/americanexpress/babel-preset-amex) to determine which modern browsers are supported. Generated using [one-app-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler) which uses [webpack](https://webpack.js.org/) under the hood. |
+| `<module-name>.browser.js.map` | Contains the [source map](https://www.html5rocks.com/en/tutorials/developertools/sourcemaps/) which has details about the origin source code. This is loaded on modern browsers to aid in debugging and it is only generated when the `process.env.NODE_ENV` is set to `development`.|
+| `<module-name>.legacy.browser.js` | This build is loaded on legacy browsers. We use [babel-preset-amex](https://github.com/americanexpress/babel-preset-amex) to determine which legacy browsers are supported. Generated using [one-app-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler) which uses [webpack](https://webpack.js.org/) under the hood.|
+| `<module-name>.legacy.browser.js.map` | Contains the [source map](https://www.html5rocks.com/en/tutorials/developertools/sourcemaps/)  which has details about the origin source code. This is loaded on legacy browsers to aid in debugging. it is only generated when the `process.env.NODE_ENV` is set to `development`.|
+| `<module-name>.node.js` | This build is loaded on `one-app` . Anything set to be server side rendered would executed once this file is loaded on one-app. |
+| `<module-name>.node.js.map` | Contains the [source map](https://www.html5rocks.com/en/tutorials/developertools/sourcemaps/)  which has details about the origin source code. This allows you to debug code running on one-app. This is only generated when the `process.env.NODE_ENV` is set to `development`. |
```

Comment:
There is an extra space between `/sourcemaps/)` and `which`

Reply:
Thanks for catching this updated c5b191e